committee-steering/governance/annual-reports.md

```diff
@@ -62,57 +62,220 @@ of being more explicit than using our upstream shorthand or abbreviations.
-What are some activities that the group does to encourage this? What programs
-are you participating in to grow contributors throughout the contributor ladder?
-- What programs do you participate in for new contributors?
-- Does the group have contributors from multiple companies/affiliations? Can end
- users/companies contribute in some way that they currently are not?
-
-##### Current initiatives and project health
-- What are initiatives that should be highlighted, lauded, shout outs, that
-your group is proud of? Currently underway? What are some of the longer tail
-projects that your group is working on?
-- Year to date KEP work: What's now stable? Beta? Alpha? Road to alpha?
-- What initiatives are you working on that aren't being tracked in KEPs?
-- What areas and/or subprojects does the group need the most help with?
-- What metrics/community health stats does your group care about and/or measure?
-Examples?     
+## Questions for report:
+
+### Special Interest Groups:
+
+```
+# $sig-name - $YYYY annual report
+
+## Current initiatives
+
+1. What work did the SIG do this year that should be highlighted?
+
+   - 
+   - 
+   - 
+
+2. What initiatives are you working on that aren't being tracked in KEPs?
+
+   - 
+   - 
+   - 
+
+3. KEP work in $YYYY (1.x, 1.y, 1.z):
+
+<!-- 
+Generated from kubernetes/enhancements kep.yaml files
+1. with SIG as owning-sig or in participating-sigs
+2. listing 1.x, 1.y, or 1.z in milestones or in latest-milestone
+-->
+
+   - Stable
+      - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.stable
+      - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.stable
+   - Beta
+      - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.beta
+      - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.beta
+   - Alpha
+      - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
+      - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
+   - Pre-alpha
+      - [$kep-number - $title](https://git.k8s.io/community/$link/README.md)
+
+## Project health
+
+1. What areas and/or subprojects does your group need the most help with?
+   Any areas with 2 or fewer OWNERs? (link to more details)
+
+   - 
+   - 
+   - 
+
+2. What metrics/community health stats does your group care about and/or measure?
+
+   - 
+   - 
+   - 
+
+3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group specifically by pointing 
+   to activities or programs that provide useful context or allow easy participation?
+
+   - 
+
+4. Does your [CONTRIBUTING.md] help **existing** contributors grow throughout the [contributor ladder]
+   by documenting any special training, requirements, or processes unique to reviewers/approvers in your group?
+
+   - 
+
+5. Does the group have contributors from multiple companies/affiliations?
```

Comment:
If we want to template this or provide a url to get the info - here is the url for sig-architecture: 
https://k8s.devstats.cncf.io/d/66/developer-activity-counts-by-companies?orgId=1&var-period_name=Last%20year&var-metric=contributions&var-repogroup_name=SIG%20Architecture&var-repo_name=kubernetes%2Fkubernetes&var-country_name=All&var-companies=All&viewPanel=1

the `var-repogroup_name=SIG%20Architecture`is the one part that would need to be adjusted for each sig

Reply:
a few things I noticed about that:
1. the "last year" period is not stable... ideally, we'd want "2021"
2. it's not clear how that dashboard intersects contributions and sigs
3. there doesn't seem to be a way to get activity across kubernetes and kubernetes-sigs

for this year, I probably won't add that link, but I'll open issues to track those things

Reply:
> a few things I noticed about that:
> 
> 1. the "last year" period is not stable... ideally, we'd want "2021"
> 2. it's not clear how that dashboard intersects contributions and sigs
> 3. there doesn't seem to be a way to get activity across kubernetes and kubernetes-sigs
> 
> for this year, I probably won't add that link, but I'll open issues to track those things

re: 1 - Devstats doesn't record that information in year form :( They do have releases which are a rough approximation when you look at them over a period. - Maybe not as a tool to report on, but a resource they can use and self report? like as a comment.

re:  2/3 - They do, but the dashboard UI is...not intuitive. The dashboard I linked is for sig-arch and across all orgs, but has the filters for repos on there as well. The "contributions" stat is issues, PRs, commits, comments etc.

Reply:
I think the template is good to go as is, but can distribute the direct link to the SIGs for their own adhoc check. Will resolve this convo 👍